include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -2287,10 +2181,9 @@ namespace internal
             gradients_quad += dim * n_q_points;
           }
 
-      if (evaluate_hessian)
+      if (evaluation_flag & EvaluationFlags::hessians)
         {
-          values_dofs                 = values_dofs_ptr;
-          constexpr unsigned int hdim = (dim * (dim + 1)) / 2;
```

Comment:
Why do you not like `hdim`? I would prefer if we would keep, so that the value does not seem so random!

Reply:
It is used only once, and in a place pretty far away. We might re-introduce it if you want, but then close to the use, even though I see not less randomness unless we choose a longer, more expressive name.